bibliopixel/animation/animation.py

```diff
@@ -115,6 +115,7 @@ def stamp():
     @contextlib.contextmanager
     def run_context(self):
         self.state = STATE.running
+        self.threading.stop_event.clear()
```

Comment:
Just by putting some debug statements in there. I quickly realized that what was happening was that after the first go through each animation would run a single frame and then go to the next. I have no idea when exactly this broke. I'd say after 3.0.5 though. I think it worked at some point when I had remote going. I don't know... fixed now :P